Script/AtomicEditor/main.ts

```diff
@@ -3,6 +3,16 @@
 /// <reference path="../TypeScript/Editor.d.ts" />
 /// <reference path="../TypeScript/AtomicWork.d.ts" />
 
-import __Editor = require("./editor/Editor");
+import Editor = require("editor/Editor");
 
-var TheEditor = new __Editor();
+class Main {
+    static Editor:Editor;
+
+    static main() {
+        Main.Editor = new Editor();
+    }
+}
+
+Main.main();
+
+export = Main;
```

Comment:
This export makes it seem like we're requiring this, however we are evaling it.  Let's remove the export = Main, eventually I would like to get main.ts required as this is still causing a Main in the global namespace as we're requiring